container/CrabV2/Auction/Auction.tsx

```diff
@@ -202,13 +214,17 @@ const AuctionDetailsHeader: React.FC<{ isAuctionLive: boolean; isSelling: boolea
           <Typography color="textSecondary">clearing price(per oSQTH)</Typography>
           <Typography textAlign="center" variant="numeric" color="primary">
             {formatBigNumber(auction.clearingPrice || '0', 18, 6)} WETH
+           <Typography textAlign="center" variant="numeric"  color="textSecondary">  ${(calculateDollarValue(convertBigNumber(auction.clearingPrice, 18), ethPrice)).toFixed(2)}  </Typography> 
+            <Typography  variant="numeric"  color="textSecondary">  {(calculateIV(oSqthPrice, nf, convertBigNumber(auction.clearingPrice, 18)) * 100).toFixed(2)}% </Typography>
```

Comment:
I think this is wrong, calculateIV takes "oSqthPrice, nf, ethPrice" as params. "clearingPrice" is the price of squeeth and not eth Price. Eth price will be the same in all the places. The only difference is the squeeth price. Let me know if it makes sense

Reply:
Fixed